TvShowTreeView.cpp compile problem on Fedora 37

**Describe the bug**
I'm trying to compile on Fedora 37 x86_64, and I get an error from g++:

```
../src/ui/small_widgets/TvShowTreeView.cpp: In member function ‘void TvShowTreeView::drawTvShowRow(QPainter*, const QStyleOptionViewItem&, const QModelIndex&) const’:
../src/ui/small_widgets/TvShowTreeView.cpp:140:5: error: invalid use of member function ‘const QFont& QWidget::font() const’ (did you forget the ‘()’ ?)
  140 |     font.setPointSize(font.pointSize() - 1);
      |     ^~~~
      |         ()
make: *** [Makefile:22022: TvShowTreeView.o] Error 1

```

I've been able to work around it by changing `font.setPointSize(font.pointSize() - 1);` to `textFont.setPointSize(textFont.pointSize() - 2);` as is done for Macintosh builds.

**To Reproduce**
Steps to reproduce the behavior:
1. Compile on Fedora 37.

**Expected behavior**
No compiler error

**Screenshots**
If applicable, add screenshots to help explain your problem.

**MediaElch Version:**
 - [X] 2.8.19-dev (nightly)
 - [ ] 2.8.18 stable
<!-- older stable versions are not supported; please update -->

**Operating System:**
 - [ ] Windows
 - [ ] macOS
 - [X] Linux (distribution: Fedora 37)

**Additional context**
Add any other context about the problem here.

--- a/src/ui/small_widgets/TvShowTreeView.cpp
+++ b/src/ui/small_widgets/TvShowTreeView.cpp
@@ -137,7 +137,7 @@ void TvShowTreeView::drawTvShowRow(QPainter* painter,
 #ifdef Q_OS_MAC
     textFont.setPointSize(textFont.pointSize() - 2);
 #else
-    font.setPointSize(font.pointSize() - 1);
+    textFont.setPointSize(textFont.pointSize() - 1);
 #endif
     textFont.setBold(false);
 

